### PR TITLE
Create 0xCUBE.md

### PR DESCRIPTION
**Pull request by 0xCUB3**
Add my journey

### CODE REVIEW

File: journeys/0xCUBE.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<p align="center">
+  <img src="images/Small-Cube.png" />
```
Comment (KendallDoesCoding) [inline journeys/0xCUBE.md:2]: image not working

image

File: journeys/0xCUBE.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+</p align="center">
+
+<h1 align="center">Hi! I'm 0xCUBE! Welcome to my profile!</h1>
```
Comment (KendallDoesCoding) [inline journeys/0xCUBE.md:5]:
```suggestion
<h1 align="center">Hi! I'm 0xCUBE! Welcome to my journey!</h1>
```

File: journeys/0xCUBE.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+<p align="center">
+ 
+ <img src="https://badges.pufler.dev/visits/0xCUB3/0xCUB3"/> 
+ <!-- <img src="https://badges.pufler.dev/years/ritik307"/> -->
+ <img src="https://badges.pufler.dev/repos/0xCUB3"/>
+ <img src="https://badges.pufler.dev/commits/yearly/0xCUB3" />
```
Comment (KendallDoesCoding) [inline journeys/0xCUBE.md:11]: images not working

image

File: journeys/0xCUBE.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+<br>
+
+<h2 align="center">My GitHub Stats</h2>
+<p align = "center">
+  <img  src = "https://github-readme-stats.vercel.app/api?username=0xCUB3&show_icons=true&theme=radical">
+  <img src = "https://github-readme-stats.vercel.app/api/top-langs/?username=0xCUB3&theme=radical">
+</p>
+<br>
+
+<h2 align="center">Social Links</h2>
```
Comment (KendallDoesCoding) [inline journeys/0xCUBE.md:40]: social links not working


image

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
Images still not working after edits.

**Comment by 0xCUB3** [comment]
hmm odd, let me see again

File: journeys/0xCUBE.md (hunk 3 above)
Comment (KendallDoesCoding) [reply journeys/0xCUBE.md:11]: This is still not working, rest is fixed.

**Review by KendallDoesCoding — APPROVED** [review]

**Pull request merged**